plugins/katello/3.14/installation/index.md

```diff
@@ -170,6 +170,68 @@ arguments. Multiple entries can be entered via
         reverse: true
         contact: hostmaster.example.com.
 
+## Tuning options
+The Foreman installer supports automatic tuning of your environment using predefined tuning profiles.  These tuning profiles are the result of a culmination of extensive learning from Foreman environments deployed at scale in large user environments.
+
+When the foreman-installer is run, it is deployed with a `default` predefined tuning profile. Other than the default tuned profile, foreman-installer supports 4 different tuning profiles:
+
+  * medium
+  * large
+  * extra-large
+  * extra-extra-large
+
+Based on your environment needs, use one of the tuning profiles (`medium`, `large`, `extra-large`, `extra-extra-large`) in the installer.  For example, `medium` profile can be applied like:
+
+```bash
+foreman-installer --tuning medium
+```
+
+To reset to the default profile:
+
+```bash
+foreman-installer --tuning default
+```
+
+Use `foreman-installer --help | grep tuning` to identify the current tuning level.
+
+Sample output for `medium` tuning:
+
+```bash
+foreman-installer --help | grep tuning
+    --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "medium")
+```
+
+Sample output for `default` tuning:
+
+```bash
+foreman-installer --help | grep tuning
+    --tuning INSTALLATION_SIZE    Tune for an installation size. Choices: default, medium, large, extra-large, extra-extra-large (default: "default")
+```
+
+**Note**
+
+- Definitions of various tuning profiles can be found in this directory `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`.  Note that `common.yaml` is always applied and the selected tuning profile (e.g., `medium`) is applied on top and takes precedence.
+- Using the `--tuning` option does not update `/etc/foreman-installer/custom-hiera.yml`, instead it directly updates the required configuration as specified in the corresponding tuning profile. You can still use `custom-hiera.yml` to override any configuration if really needed.
+- If you had already used `custom-hiera.yml` and starting to use the tuned profiles, you may want to review the definition of tuned profiles (`/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml` and `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`) and remove the duplicated configuration entries from your `custom-hiera.yml`.
+- You can also optionally use `foreman-installer --noop` to run the installer in a test mode and identify what configurations will be changed before actually running the installer.
```

Comment:
To me it's clear that this can also be combined with `--tuning <profile>` and I also know that `--noop` implies not saving the answers (including tuning). Is that also clear for the average user?

Reply:
ah good point, I will update this.

Reply:
done